src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -3029,6 +3029,11 @@ public static void createMissingStaticSitePins(BELPin belPin, SiteInst si, Cell
                     net = si.getDesign().getGndNet();
                 }
             }
+            String belName = belPin.getBELName();
+            if (DesignTools.isBELALut(belName) && si.getCell(belName.replace('6', '5')) == null) {
```

Comment:
Should we use `belPin.getBEL().isLUT()` over `DesignTools.isBELALut()`?

Reply:
Should we consider deprecating the latter?

Reply:
Let's leave it then.